tests/appsec/api_security/test_schemas.py

```diff
@@ -276,6 +276,10 @@ def setup_request_method(self):
             data={"test_int": 1, "test_str": "anything", "test_bool": True, "test_float": 1.5234},
         )
 
+    @missing_feature(
+        context.library < "nodejs@5.57.0" and context.weblog_variant == "fastify",
+        reason="Parsing response body not supported yet",
```

Comment:
we never parse the response body, we only collect it
```suggestion
        reason="Response body not supported yet",
```